src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -124,16 +123,9 @@ _ResolveAssemblies MSBuild target.
     </ItemGroup>
 
     <ItemGroup>
-      <_ResolvedArchiveDSOStub Include="@(ResolvedFileToPublish)" Condition=" '%(ResolvedFileToPublish.Filename)%(ResolvedFileToPublish.Extension)' == 'libarchive-dso-stub.so' " />
       <ResolvedFileToPublish Remove="@(_ResolvedArchiveDSOStub)" />
```

Comment:
Should `libarchive-dso-stub.so` actually go in an MSBuild pack?

If not, should we just remove it from `RuntimeLists.xml`, so that we don't have to remove it from `@(ResolvedFileToPublish)`? It would just never be in the list.

Reply:
We have to ship it and we need to be able to read it (one copy per each supported arch), it doesn't really matter where it is otherwise.

Reply:
It would be simpler, definitely, but were can we put it so that it is shipped and in a location we can easily discover from within XABT?